src/statick_web/plugins/web_tool_plugins/htmllint_tool_plugin.py

```diff
@@ -37,15 +39,21 @@ def scan(self, package, level):  # pylint: disable=too-many-locals
         for src in package["html_src"]:
             try:
                 exe = [tool_bin] + flags + [src]
-                output = subprocess.check_output(exe,
-                                                 stderr=subprocess.STDOUT,
-                                                 universal_newlines=True)
+                output = subprocess.check_output(
+                    exe, stderr=subprocess.STDOUT, universal_newlines=True
+                )
                 total_output.append(output)
 
             except subprocess.CalledProcessError as ex:
-                if ex.returncode not in [1, 2]:  # tool returns 1 upon warnings and 2 upon errors
-                    print("{} failed! Returncode = {}".
-                          format(tool_bin, str(ex.returncode)))
+                if ex.returncode not in [
+                    1,
+                    2,
+                ]:  # tool returns 1 upon warnings and 2 upon errors
```

Comment:
Nit: if you move the comment to the line above the code then the code will likely stay on a single line.